ideascube/conf/base.py

```diff
@@ -298,6 +298,14 @@
         'description': _('Change the server name.'),
         'fa': 'info-circle',
     },
+    {
+        'is_staff': True,
+        'category': 'manage',
+        'url': 'server:cards',
+        'title': _('Cards'),
+        'description': _('Manage displayed cards'),
```

Comment:
I wonder if « cards » isn't internal jargon. Is this something users generally understand in the field?

To me at least, the only place I've ever heard of "cards" are in the source code and the settings files, so if I wasn't already familiar with these I don't think I would understand.

Maybe at least change it to « Manage cards displayed on the home page », which could be clearer?

Or maybe we should just talk about "apps", as that's what users tend to be more familiar with... But then my view might be biased by the fact my sample for "users" only includes people in developed countries with access to an app ecosystem on their computer/smartphone, and as such our target users might not know of "apps" any better than they do of "cards", so that's a moot point.